flipperdevices/Flipper-Android-App
# Fix infrared keys list padding

**Background**

On infrared screen, when enabled android buttons gesture navigation, the last button located beyound bottom navigation

**Changes**

- Add element into list with bottom nav panel padding. The padding not added to column itself to save transparent display of navigation panel.

**Test plan**
- Enabled buttons panel navigation in device settings
- Open infrared editor with more keys than screen height
- Scroll down
- See the last element now not located beyound bottom navigation
